x/ecocredit/base/keeper/msg_remove_allowed_bridge_chain_test.go

```diff
@@ -0,0 +1,73 @@
+package keeper
+
+import (
+	"encoding/json"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+
+	types "github.com/regen-network/regen-ledger/x/ecocredit/base/types/v1"
+)
+
+type removeAllowedBridgeChainSuite struct {
+	*baseSuite
+	err error
+}
+
+func TestRemoveAllowedBridgeChain(t *testing.T) {
+	gocuke.NewRunner(t, &removeAllowedBridgeChainSuite{}).Path("./features/msg_remove_allowed_bridge_chain.feature").Run()
+}
+
+func (s *removeAllowedBridgeChainSuite) Before(t gocuke.TestingT) {
+	s.baseSuite = setupBase(t)
+}
+
+func (s *removeAllowedBridgeChainSuite) ExpectChainNameToNotExist(a string) {
+	chainName := a
+	found, err := s.stateStore.AllowedBridgeChainsTable().Has(s.ctx, chainName)
+	require.NoError(s.t, err)
+	require.False(s.t, found)
+}
+
+func (s *removeAllowedBridgeChainSuite) ExpectTheError(a string) {
+	require.EqualError(s.t, s.err, a)
+}
+
+func (s *removeAllowedBridgeChainSuite) TheAuthorityAddress(a string) {
+	addr, err := sdk.AccAddressFromBech32(a)
+	require.NoError(s.t, err)
+	s.authority = addr
+}
```

Comment:
Same as above. Might be nice to use the same ordering as we do in other gocuke tests and list `Given` step functions, then `When` step functions, then `Then` step functions.